doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -709,6 +709,31 @@ Note also that in the absence of a ``as ident``, ``as strict pattern`` or
 ``as pattern`` :n:`@syntax_modifier`\s, the default is to consider sub-expressions occurring
 in binding position and parsed as terms to be ``as ident``.
 
+Binders bound in the notation and parsed as general binders
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+It is also possible to rely on Coq's own syntax of parsers using the
+`binder` modifier as follows:
+
+.. coqtop:: in
+
+   Notation "'myforall' p , [ P , Q ] " := (forall p, P -> Q)
+     (at level 200, p binder).
+
+In this case, all of `@ident`, `{@ident}`, `[@ident]`, `@ident:@type`,
+`{@ident:@type}`, `[@ident:@type]`, `'pattern` can be used in place of
+the corresponding notation variable. In particular, the binding can be
+used to declare implicit arguments:
+
+.. coqtop:: all
+
+   Check fun (f : myforall {a}, [a=0, Prop]) => f eq_refl.
+   Check myforall '((x,y):nat*nat), [ x = y, True ].
+
+By using instead `closed binder`, parentheses, the same list of syntax
```

Comment:
I don't understand this ", parentheses,"

Reply:
A typo, sorry. I reworded the paragraph, as well as the first one.